static/cypress/integration/utils.ts

```diff
@@ -66,6 +66,14 @@ export function intercept_get_image() {
   ).as("get_image");
 }
 
+export function intercept_get_image_with_failure() {
+  cy.intercept(
+    "GET",
+    `${Cypress.env("API_URL")}/tasks/ecg-qrs-classification-physiodb/sloth.jpg`,
+    { statusCode: 404 }
+  ).as("get_image");
+}
+
```

Comment:
I don't see how this belongs here? 🤔

Reply:
Ah good question: How do we want to organize utility functions? Because they are needed for the component tests and integration tests. We could put the component tests in the cypress folder too, or move the utils out of the integration folder